actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -1255,11 +1255,8 @@ public boolean canActivateActor(final String interfaceName)
     {
         if (getState() != NodeCapabilities.NodeState.RUNNING)
         {
-            // todo, improve this
-            if (hosting.getServerNodes().size() > 1)
```

Comment:
@DanielSperry this is a bit of a wtf line in my opinion, if the node is shuts down in the middle of Hosting select node I can see where it might be possible for an incoming request to be forwarded to a node after it has started to shut down which would either result in a timeout or the Actor being killed in a few seconds?